### PR TITLE
Add spec endpoint to dotnet proxy

### PR DESCRIPTION
**Pull request by emmawillis**

### CODE REVIEW

File: proxies/dotnet/dotnet.sln [hunk 1]
```diff
@@ -0,0 +1,25 @@
+﻿
```
[the remark this answers is not in the document]
Comment (emmawillis) [reply proxies/dotnet/dotnet.sln]: oh true, fixed!

File: proxies/dotnet/Controllers/Spec.cs [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        _logger = logger;
+    }
+
+    [HttpGet]
+    public string Get()
+    {
+        return JsonConvert.SerializeObject(new {
+            name = "Dotnet",
+            version = "",
+            capabilities = new List<string>(){"EdgeDB", "LocalBucketing"}
```
Comment (ajwootto) [inline proxies/dotnet/Controllers/Spec.cs:24]: shouldnt this also list cloud?

Comment (emmawillis) [reply proxies/dotnet/Controllers/Spec.cs]: oops, fixed!

File: proxies/dotnet/dotnet.csproj [hunk 3]
```diff
@@ -6,6 +6,8 @@
     <ImplicitUsings>enable</ImplicitUsings>
   </PropertyGroup>
 
+  <PropertyGroup Condition=" '$(RunConfiguration)' == 'https' " />
```
Comment (ajwootto) [inline proxies/dotnet/dotnet.csproj:9]: is this to fix the https redirect warning? is it gonna redirect all requests to https after this change? not sure that will work in docker

Comment (emmawillis) [reply proxies/dotnet/dotnet.csproj]: i'm not sure what this does tbh, it was auto generated when i ran it in vs code so i assumed it was needed to run dotnet run directly - but it works fine without it so i'll remove

**Review by ajwootto — APPROVED** [review]

**Review by jsalaber — APPROVED** [review]

**Pull request merged**